### PR TITLE
Add support for Scala 2.13

### PR DESCRIPTION
**Pull request by julienrf**
Support for 2.11 has been disabled because scalacss does not support it.

There is a problem wth XML literals. Has anyone an idea of how to support them?

### CODE REVIEW

**Comment by julienrf** [comment]
Fixes #109

File: examples/target/scala-2.13/examples-opt.js [hunk 1]
```diff
@@ -0,0 +1,1214 @@
+(function(){'use strict';
```
Comment (esarbe) [inline examples/target/scala-2.13/examples-opt.js]: Can we drop this file?

**Comment by esarbe** [comment]
Can we get this in? Seems alright, no?

File: examples/target/scala-2.13/examples-opt.js (hunk 1 above)
Comment (OlivierBlanvillain) [reply examples/target/scala-2.13/examples-opt.js]: It's used for https://olivierblanvillain.github.io/monadic-html/examples/

**Comment by saksmt** [comment]
Any progress on this?

File: examples/target/scala-2.13/examples-opt.js (hunk 1 above)
Comment (esarbe) [reply examples/target/scala-2.13/examples-opt.js]: Better keep it then.

**Comment by OlivierBlanvillain** [comment]
It's out: https://repo1.maven.org/maven2/in/nvilla/monadic-html_sjs0.6_2.12/0.4.0/

**Comment by esarbe** [comment]
Awesome!

**Pull request merged**